docs/Makefile

```diff
@@ -35,7 +35,7 @@ attributes:
 		>> "${managed_partials}/attributes.adoc"
 	echo ":recommended-java-version: 11" \
 		>> "${managed_partials}/attributes.adoc"
-	echo ":java_minimum_sdk_version: 0.7.0-beta.19" \
+	echo ":java_minimum_sdk_version: 0.7.0" \
```

Comment:
is this something we need to push to `current/docs` to get it out?

Reply:
Let's push it during the next release. Johan (or you) created the PR for the main docs already so we got that covered there. Once this is in the `current/docs` branch, we can make the update in the main docs repo to use this version number rather than what it currently has.

Reply:
I see. We should improve on the Releasing.md so that all steps are clear.

Reply:
Great idea! I'll submit a new PR for that.